schema/NPSSchema.mcf

```diff
@@ -0,0 +1,187 @@
+descriptionUrl: "https://www.bjs.gov/index.cfm?tv=tdtp&tid=1"
+
+Node: dcid:prisonSentenceStatus
+name: "prisonSentenceStatus"
+typeOf: schema:Property
+rangeIncludes: dcs:PrisonSentenceStatusEnum
+domainIncludes: dcs:Person dcs:IncarcerationEvent
+description: "The status of one's prison sentence"
+
+Node: dcid:PrisonSentenceStatusEnum
+name: "PrisonSentenceStatusEnum"
+typeOf: schema:Class
+subClassOf: schema:Enumeration
+description: "An enumeration of prison sentence statuses"
+
+Node: dcid:Unsentenced
+name: "Unsentenced"
+typeOf: dcs:PrisonSentenceStatusEnum
+description: "Indicates state of not having received a prison sentence"
+
+Node: dcid:Sentenced
+name: "Sentenced"
+typeOf: dcs:PrisonSentenceStatusEnum
+description: "Indicates state of having received a prison sentence"
+
+Node: dcid:maxPrisonSentence
+name: "maxPrisonSentence"
+typeOf: schema:Property
+rangeIncludes: dcs:QuantityRange
+domainIncludes: dcs:Person dcs:IncarcerationEvent
+
+Node: dcid:NPSMeasurementBasisEnum
+name: "NPSMeasurementBasisEnum"
+typeOf: schema:Class
+subClassOf: schema:Enumeration
+description: "Enumeration for the basis of measurement of the prisoner statistic"
+
+Node: dcid:MeasuredBasedOnCustody
+name: "MeasuredBasedOnCustody"
+typeOf: dcs:NPSMeasurementBasisEnum
+description: "Measurement based on those in custody of state or federal correctional officials"
+
+Node: dcid:MeasuredBasedOnJurisdiction
+name: "MeasuredBasedOnJurisdiction"
+typeOf: dcs:NPSMeasurementBasisEnum
+description: "Measurement based on those under the jurisdiction of the state meaning all state and federal inmates held in a public or private prison and those held in jail facilities either physically located inside or outside of the state of legal responsibility, and other inmates who may be temporarily out to court or in transit from the jurisdiction of legal authority to the custody of a confinement facility outside that jurisdiction"
+descriptionUrl: “https://www.bjs.gov/index.cfm?ty=tdtp&tid=1”
+
+Node: dcid:NPSCauseOfDeathEnum
+name: "NPSCauseOfDeathEnum"
+typeOf: schema:Class
+subClassOf: schema:Enumeration
+description: "Enumeration for NPS measured causes of death"
+
+Node: dcid:JudicialExecution
+name: "JudicialExecution"
+typeOf: dcs:NPSCauseOfDeathEnum
+description: "Execution for committing a crime"
+
+Node: dcid:IllnessOrNaturalCause
+name: "IllnessOrNaturalCause"
+typeOf: dcs:NPSCauseOfDeathEnum
+description: "Illness or other natural cause of death excluding AIDS"
+
+Node: dcid:AIDS
+name: "AIDS"
+typeOf: dcs:NPSCauseOfDeathEnum
+description: "AIDS"
+
+Node: dcid:Suicide
+name: "Suicide"
+typeOf: dcs:NPSCauseOfDeathEnum
+description: "Suicide"
+
+Node: dcid:AccidentalInjuryToSelf
+name: "AccidentalInjuryToSelf"
+typeOf: dcs:NPSCauseOfDeathEnum
+description: "Accidentally caused injury to oneself"
```

Comment:
I'm not too sure about NPSCauseOfDeathEnum and instances. @tjann might have opinions here.

There is overlap with ICD10 codes (aids, suicide, homicide, accidents): http://dph1.adph.state.al.us/csc/vs/Help/Dthstatrel1/Cause_of_Death_ICD_Codes.htm.

For others, ok if you define new enums, but perhaps call the class CausesOfDeathEnum.

Reply:
Looking at the ICD10 codes in our schema, I can't seem to find aids, I only found hiv

Reply:
I think that is ok. AIDS is a special (late stage) for of HIV